### DISCUSSION TITLE
占位符热更新失效

### DISCUSSION THREAD

**Discussion by dong-chao1002** [apolloconfig/apollo#4909] [Q&A]
场景：将application.properties的数据库等配置放在Apollo上，但是将Apollo上的配置修改后，还是不能更新，如果故意将数据库信息改错，应用还能可以正常访问数据库，在启动项目时，出现[ INFO ] [2023-06-21 10:20:23.697] com.ctrip.framework.apollo.internals.ConfigServiceLocator [122] [main]- Located config services from apollo.config-service configuration: http://1.1.1.1:8097, will not refresh config services from remote meta service! 烦请各位大佬解决一下，另外在application.peoperties上要写上apollo.config-service=http://1.1.1.1:8097才能正常启动连接Apollo。

**Comment by nobodyiam** [comment]
see https://github.com/apolloconfig/apollo-use-cases/tree/master/dynamic-datasource
